src/shared/ui/survey/TimeRangeItem.tsx

```diff
@@ -1,11 +1,13 @@
 import { FC } from 'react';
 
+import { format } from 'date-fns';
+
 import { colors } from '@app/shared/lib';
 import { YStack, DateTimePicker, AlarmIcon, BedIcon } from '@shared/ui';
 
 type TimeRangeValue = {
-  from: string;
-  to: string;
+  endTime: string;
```

Comment:
We have the data-type HourMinute. 
Can we refactor this component in order to use exactly this data-type?